docker/hisat2/Dockerfile

```diff
@@ -10,7 +10,25 @@ WORKDIR /opt/tools
 
```

Comment:
Could we get a comment in each of the dockers that lists the commands they're supposed to be able to run and how to access them? 

i.e. `hisat2` if it's accessible on the path, or `/opt/tools/hisat2/` if not.

Reply:
add one line comment